gap/PolygonalComplexes/animating.gi

```diff
@@ -797,170 +806,302 @@ InstallMethod( GetCircleColours,
+                Append(coordinateStringB, ",");
+                Append(coordinateStringB, String(GetVertexCoordinates3DNC(surface, edgeVertexB, printRecord)[2]));
+                Append(coordinateStringB, ",");
+                Append(coordinateStringB, String(GetVertexCoordinates3DNC(surface, edgeVertexB, printRecord)[3]));
+                Append(coordinateStringB, ",");
+
+                AppendTo(output, "\t \t \t",coordinateStringA,"\n",coordinateStringB,"\n \n");
+            # fi;
+        od;
 
-                # Add Edges to scenario
-                template := __SIMPLICIAL_ReadTemplateFromFile("JS_init_edges.html.template");
-                AppendTo( output, template );
-                if not IsBound(printRecord.edges) then
-                    printRecord := CalculateParametersOfEdges(surface, printRecord);
-                fi;
-                if calculate then
-                    printRecord := CalculateParametersOfEdges(surface, printRecord);
-                fi;
-                for edge in edges do
-                    if IsEdgeActive(surface, edge, printRecord) then
-                        parametersOfEdge := printRecord.edges[edge];
-                        colour := GetEdgeColour(surface, edge, printRecord);
-                        if not StartsWith(colour, "0x") then
-                            colour := Concatenation("\"", colour, "\"");
-                        fi;
-                        if IsBound(printRecord.edgeThickness) then
-	                        AppendTo(output, "\t\tvar edge = Edge(", parametersOfEdge[2], ",", printRecord.edgeThickness,",", parametersOfEdge[1][1], ", ",
-	                            parametersOfEdge[1][2], ", ", parametersOfEdge[1][3], ", ", parametersOfEdge[3][1], ", ",
-	                            parametersOfEdge[3][2], ", ", parametersOfEdge[3][3], ", ", colour, ");\n");
-	                        AppendTo(output, "\t\tobj.add(edge);\n");
-	                    else
-	                    	AppendTo(output, "\t\tvar edge = Edge(", parametersOfEdge[2], ", 0.01, ", parametersOfEdge[1][1], ", ",
-	                            parametersOfEdge[1][2], ", ", parametersOfEdge[1][3], ", ", parametersOfEdge[3][1], ", ",
-	                            parametersOfEdge[3][2], ", ", parametersOfEdge[3][3], ", ", colour, ");\n");
-	                        AppendTo(output, "\t\tobj.add(edge);\n");
-	                    fi;
-                    fi;
-                od;
+        AppendTo(output, "\t \t \t]);");
+        AppendTo(output, """
+            const edgeGeometry""",i,""" = new THREE.BufferGeometry();
+            edgeGeometry""",i,""".setAttribute( 'position', new THREE.BufferAttribute( edges""",i,""", 3 ) );
+
+            const edgeLine""",i,""" = new THREE.LineSegments( edgeGeometry""",i,""", edgeMaterial""",i,""" );
+            edgeRoot.add(edgeLine""",i,""");
+        """);
+        
+    od;
+    # for i in [1..Length(faces)] do
```

Comment:
Can this commented out part be removed?

Reply:
Removed in [e5ec4ce](https://github.com/gap-packages/SimplicialSurfaces/pull/248/commits/e5ec4ce7bd21dde09732f1ffb3b2b66b7d9f3c2f)